test/src/test/kotlin/graphql/nadel/tests/next/fixtures/partition/PartitionWithConflictingRoutingValuesTestSnapshot.kt

```diff
@@ -21,45 +21,7 @@ private suspend fun main() {
 @Suppress("unused")
 public class PartitionWithConflictingRoutingValuesTestSnapshot : TestSnapshot() {
     override val calls: List<ExpectedServiceCall> = listOf(
-            ExpectedServiceCall(
```

Comment:
the service call on this test is no longer executed, since we're throwing an error on the `transformField` method